conda_smithy/feedstock_content/ci_support/checkout_code.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+
```

Comment:
This is being tracked in issue ( https://github.com/conda-forge/conda-smithy/issues/316 ).